README.md

```diff
@@ -123,8 +123,9 @@ query MyQuery {
       headerImage {
         localFile {
           extension
+          url
```

Comment:
I have 'error  Cannot query field "url" on type "File".' when using this example

![Desktop-screenshot (51)](https://user-images.githubusercontent.com/645716/142991435-914d5cde-287c-4421-9c55-20695e0c6edd.png)

Reply:
fixed